### PR TITLE
Fixes lint errors #111

### PR DESCRIPTION
**Pull request by hudochenkov**
I've fixed lint errors. Also i've not converted gulpfile to gulpfile.babel because I have no experience with this, so I simply add it to ignore for ESLint for now.

Also ESLint was going to lint design/node_modules, so I've fixed this too.

### CODE REVIEW

**Comment by mxstbr** [comment]
Can you try renaming the Gulpfile to Gulpfile.babel.js and see if that works? Thanks!

**Comment by hudochenkov** [comment]
@mxstbr it works! I shouldn't read articles about this and just need to rename the file :)

File: .gitignore [hunk 1]
```diff
@@ -4,7 +4,8 @@
 # node cache/logs
 npm-debug.log
 node_modules
+design/node_modules
 
 # statinamic builded files
-/dist
+dist
```
Comment (hudochenkov) [inline .gitignore]: Made this change because ESLint report an error on builded files.

Comment (mxstbr) [inline .gitignore]: Sounds good, those are minified so it makes sense that the linter would error that.

**Comment by mxstbr** [comment]
LGTM

Good job @hudochenkov, thanks for this PR!

**Comment by marcustisater** [comment]
from now on, we have to be strict that linting should pass before merged into master.

**Pull request merged**